test/util/ChartUtils/parseScale.spec.ts

```diff
@@ -0,0 +1,197 @@
+import {
+  scaleBand,
+  scaleDiverging,
+  scaleDivergingLog,
+  scaleDivergingPow,
+  scaleDivergingSqrt,
+  scaleDivergingSymlog,
+  scaleIdentity,
+  scaleLinear,
+  scaleLog,
+  scaleOrdinal,
+  scalePoint,
+  scalePow,
+  scaleQuantile,
+  scaleQuantize,
+  scaleRadial,
+  scaleSequential,
+  scaleSequentialLog,
+  scaleSequentialPow,
+  scaleSequentialQuantile,
+  scaleSequentialSqrt,
+  scaleSequentialSymlog,
+  scaleSqrt,
+  scaleThreshold,
+  scaleTime,
+  scaleUtc,
+} from 'victory-vendor/d3-scale';
+import { parseScale } from '../../../src/util/ChartUtils';
+
+describe('parseScale', () => {
+  describe('d3-scale function string', () => {
+    test.each([
+      ['linear', 'scaleLinear', scaleLinear()],
+      ['Linear', 'scaleLinear', scaleLinear()],
+      ['pow', 'scalePow', scalePow()],
+      ['sqrt', 'scaleSqrt', scaleSqrt()],
+      ['log', 'scaleLog', scaleLog()],
+      ['identity', 'scaleIdentity', scaleIdentity()],
+      ['radial', 'scaleRadial', scaleRadial()],
+      ['time', 'scaleTime', scaleTime()],
+      ['utc', 'scaleUtc', scaleUtc()],
+      ['sequential', 'scaleSequential', scaleSequential()],
+      ['sequentialLog', 'scaleSequentialLog', scaleSequentialLog()],
+      ['sequentialPow', 'scaleSequentialPow', scaleSequentialPow()],
+      ['sequentialSqrt', 'scaleSequentialSqrt', scaleSequentialSqrt()],
+      ['sequentialSymlog', 'scaleSequentialSymlog', scaleSequentialSymlog()],
+      ['sequentialQuantile', 'scaleSequentialQuantile', scaleSequentialQuantile()],
+      ['diverging', 'scaleDiverging', scaleDiverging()],
+      ['divergingLog', 'scaleDivergingLog', scaleDivergingLog()],
+      ['divergingPow', 'scaleDivergingPow', scaleDivergingPow()],
+      ['divergingSqrt', 'scaleDivergingSqrt', scaleDivergingSqrt()],
+      ['divergingSymlog', 'scaleDivergingSymlog', scaleDivergingSymlog()],
+      ['quantize', 'scaleQuantize', scaleQuantize()],
+      ['quantile', 'scaleQuantile', scaleQuantile()],
+      ['threshold', 'scaleThreshold', scaleThreshold()],
+      ['ordinal', 'scaleOrdinal', scaleOrdinal()],
+      ['band', 'scaleBand', scaleBand()],
+      ['point', 'scalePoint', scalePoint()],
+    ])('with input %s should return %s', (input, expectedName, expectedFn) => {
+      const result = parseScale({ scale: input });
+      expect(result).toEqual({
+        realScaleType: expectedName,
+        scale: expect.any(Function),
```

Comment:
I am in doubt as to the value of checking whether the scale is a function, but I do see great value in adding these tests to improve documentation. So lets iterate! 🚀